packages/shared/src/components/cards/v1/RaisedLabel.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { SimpleTooltip } from '../../tooltips/SimpleTooltip';
+import classed from '../../../lib/classed';
+
+export enum RaisedLabelType {
+  Hot = 'Hot',
+  Pinned = 'Pinned',
+  Beta = 'Beta',
+}
+
+const typeToClassName: Record<RaisedLabelType, string> = {
+  [RaisedLabelType.Hot]: 'bg-theme-status-error',
+  [RaisedLabelType.Pinned]: 'bg-theme-bg-bun',
+  [RaisedLabelType.Beta]: 'bg-cabbage-40',
+};
+
+export interface RaisedLabelProps {
+  type: RaisedLabelType;
+  description?: string;
+  className?: string | undefined;
+}
+
+const Spacer = classed(
+  'div',
+  'mt-[7px] w-2 h-1 bg-theme-bg-primary  group-hover:bg-theme-bg-secondary group-focus:bg-theme-bg-secondary group-focus:mt-1.5',
```

Comment:
7px can't be right, our rule of thumbs is that's it's dividable
CC: @sshanzel

Reply:
7px is exactly right for what I want to achieve with that spacer. [See the comment here](https://github.com/dailydotdev/apps/pull/2488#discussion_r1444833110).

Reply:
Yeah, usually, the issue with the design apps is due to the borders offsetting the value.